spacy_llm/tasks/rel.py

```diff
@@ -1,7 +1,13 @@
 from typing import Callable, Dict, Iterable, List, Optional, Tuple, Type, Union
 
 import jinja2
-from pydantic import BaseModel, Field, ValidationError, validator
+
+try:
+    from pydantic.v1 import validator
+except ImportError:
+    from pydantic import validator
+
+from pydantic import BaseModel, Field, ValidationError
```

Comment:
I think you want to move all these imports into the `try`/`except` block because they might not interact like you'd expect? (I am not 100% sure, but it could be the source of the warnings. I don't understand why it's only in some python versions, though.)

Reply:
I'm also wondering why it's only on Mac with Python 3.8 :upside_down_face: Done in https://github.com/explosion/spacy-llm/pull/242/commits/3bcf2a8339439cc268cee055887c3fec3aa5b6c4.

Reply:
Done in https://github.com/explosion/spacy-llm/pull/242/commits/19410177bd8630c55ed1f58529cf4c9091fb3a02. Seems to get rid of the warnings indeed - good call, thank you!